appserver/payara-appserver-modules/hazelcast-ejb-timer/src/main/java/fish/payara/ejb/timer/hazelcast/HazelcastTimerStore.java

```diff
@@ -1193,4 +1157,37 @@ private boolean restoreEJBTimers() {
         return rc;
     }
 
+    @Override
+    public void memberAdded(MemberEvent event) {
+        //do nothing
+    }
+
+    @Override
+    public void memberRemoved(MemberEvent event) {
+        ILock hazelcastLock = Globals.getDefaultBaseServiceLocator().getService(HazelcastCore.class).getInstance().getLock("EJB-TIMER-LOCK");
```

Comment:
Isn't there a field holding the `HazelcastCore` that was passed to the constructor?